definitions/ext-rclusterhost/definition.yml

```diff
@@ -0,0 +1,24 @@
+domain: EXT
+type: HOST
```

Comment:
This doesn't match the file name like other examples do. Clearly, this is because ext host already exists. As of today, I'm not clear if that is problematic - it depends on two internal decisions. I'll brief them here for transparency

1. we're debating if or not one entity type should have multiple agents (1:1 or N:1 agent:entityType) We should have an answer on this soon (my guess is about a week.. - I'll keep you posted)
2. In the event we permit N:1 agent:entityTypes, we need to decide if this is the format we want to accept (likely it's ok, but need agreement)

Reply:
So we came to the agreement that N:1 agent:entityTypes is desirable!

As for format, it might make more sense to have all ex-host entity types together. As a suggestion, how about a new agent specific definition file under directory ext-host? eg ext-host/rcluster/{GM/SM/definition}  @srvaroa @IreneMLC opinions?

Reply:
Since we want to make this its own type it's fine in a separate directory, but the type in L2 should be `RHOST`.